index.d.ts

```diff
@@ -2966,6 +3003,7 @@ declare namespace Eris {
     getNitroStickerPacks(): Promise<{ sticker_packs: StickerPack[] }>;
     getOAuthApplication(appID?: string): Promise<OAuthApplicationInfo>;
     getPins(channelID: string): Promise<Message[]>;
+    getPollAnswerVoters(channelID: string, messageID: string, answerID: string, options?: GetRESTGuildMembersOptions): Promise<User[]>;
```

Comment:
> `options?: GetRESTGuildMembersOptions`

@bsian03 This is temporary until you tell me what you want me to do w/ it. These options expect the same options. I can either create a duplicate options interface with a different name (seems p dumb), or you can give me something to rename `GetRESTGuildMembersOptions` to so it doesn't look odd using that interface name in this context, or I can just leave it as is

Reply:
I'm going to say create a duplicate interface (non extending) as all other instances of interfaces containing a combination of after/before/limit etc do not extend each other, instead are listed separately in each relevant interface

Reply:
Okidoki. Done